web_poet/fields.py

```diff
@@ -94,24 +94,27 @@ def __get__(self, instance, owner=None):
             return self.unbound_method(instance)
 
         @staticmethod
-        def _process(value):
-            for processor in out:
-                value = processor(value)
+        def _process(value, page_object):
+            if hasattr(page_object, "validate_input"):
+                validation_item = page_object._validate_input()
+                if validation_item is not None:
+                    return getattr(validation_item, method.__name__)
```

Comment:
What do you think about adding a section to fields.rst documentation, which mentions how fields work with validate_item? For example, we can explan the following:

1. Field value may be changed to the one returned by validate_item
2. It is ok to use fields in validate_item, though they must be sync